Lesson1/MyPlayground.playground/Contents.swift

```diff
@@ -1,2 +1,73 @@
-import UIKit
+var inputData: [Int] = [1, 2, 4, 1]
 
+/// The mean for an array of integers
+/// - Parameter inputData:The array of integers with an amount greater than 2
+/// - Returns: The mean; otherwise, nil.
+func mean(for inputData: [Int]) -> Double? {
+    if inputData.count < 2 { return nil }
+    var sum = 0
+    let count = inputData.count
+    for element in inputData {
+        sum += element
+    }
+    let mean = Double(sum) / Double(count)
+    return mean
+}
+
+/// The median for an array of integers
+/// - Parameter inputData: The array of integers with an amount greater than 2
+/// - Returns: The median; otherwise, nil.
+func median(for inputData: [Int]) -> Double? {
+    if inputData.count < 2 { return nil }
+    var array = inputData
+    var median: Double?
+    for j in 1..<array.count {
```

Comment:
I suggest to move the sorting code into a separate function.

Reply:
used the method sort(by:)